### DISCUSSION TITLE
[Questions] Online Playground Difference from Actual Usage

### DISCUSSION THREAD

**Discussion by huangzongzhao** [stepci/stepci#135] [Q&A] [answered]
Hi, stepci community.

When I try the online playground, I can get some performance detailed infos like  dns/tcp/tls:
<img width="1345" alt="WechatIMG1523" src="https://github.com/stepci/stepci/assets/46254633/41ba880f-1668-41c8-9413-f6fa6c3becf5">

But when I try by stepci CLI, can't get:
<img width="649" alt="WechatIMG1524" src="https://github.com/stepci/stepci/assets/46254633/38e30c35-b272-4e30-a79a-65b66e126ea9">

**Questions: Is this expected? Or am I using stepci CLI incorrectly?**

**Comment by mishushakov** [comment, marked as answer]
Yes, this is expected. There's not much space in the CLI to put that waterfall in

**Comment by sebastianwis** [comment]
Hey @huangzongzhao 
Thank you for your contribution here! 
It would be amazing to welcome you to our Discord if you haven't joined it yet: 
https://discord.com/invite/KqJJzJ3BTu
This is where we keep users up to date with updates, features and bug resolutions.
